test/blocks/table/table.test.js

```diff
@@ -0,0 +1,94 @@
+import { readFile, sendMouse, sendKeys } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { delay, waitForElement } from '../../helpers/waitfor.js';
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/table/table.js');
+
+describe('table and tablemetadata', () => {
+  beforeEach(() => {
+    const tables = document.querySelectorAll('.table');
+    tables.forEach((t) => init(t));
+    window.dispatchEvent(new Event('milo:icons:loaded'));
+  });
+
+  describe('standard table', () => {
+    const table = document.querySelector('.table');
+
+    it('row-heading', () => {
+      expect(table.querySelector('.row-heading')).to.exist;
+    });
+
+    it('expand icon event by mouse click', () => {
```

Comment:
Nit: Tests should start with a verb in the correct tense and make sense as a sentence
```suggestion
    it('adds the row heading', () => {
      expect(table.querySelector('.row-heading')).to.exist;
    });

    it('expands on icon mouse click', () => {
```